feat: add user tag (#243)
erc7824/nitrolite: clearnode/docs/API.md

This method allows a user to transfer assets from their unified balance to another account. The user must have sufficient funds for each asset being transferred. The operation will fail if any of the specified assets have insufficient funds.

> User may specify the `destination` (wallet address) or `destination_tag` (user tag) to identify the recipient. If both are provided, `destination` takes precedence.

Reviewer: ```suggestion
User may specify the `destination` (wallet address) or `destination_tag` (user tag) to identify the recipient. `destination` is used if and only if the `destination_tag` field is empty.
```

  }], 1619123456789],
  "sig": ["0x9876fedcba..."]
}

// OR

{
  "req": [1, "transfer", [{

>     "destination_tag": "UX123D8C", // User tag

Reviewer: If we decide to go with userTag instead of tag, then this should be changed into something like:

```suggestion
    "destination_usertag": "UX123D8C", // User tag
```
